taichi/aot/module_data.h

```diff
@@ -36,7 +36,22 @@ struct CompiledOffloadedTask {
   std::string source_path;
   int gpu_block_size{0};
 
-  TI_IO_DEF(type, range_hint, name, source_path, gpu_block_size);
+  // TODO: Vulkan backend supports multiple root buffers, update this once
+  // support multiple roots in AOT.
+  int root_buffer_bind_id{-1};
```

Comment:
Should probably just look at Vulkan's buffer binds struct (a vector of tuple `<buffer, bind_id>`